app/javascript/dashboard/routes/dashboard/settings/inbox/settingsPage/ConfigurationPage.vue

```diff
@@ -161,9 +192,50 @@ export default {
         await this.$store.dispatch('inboxes/updateInbox', payload);
         this.showAlert(this.$t('INBOX_MGMT.EDIT.API.SUCCESS_MESSAGE'));
       } catch (error) {
+        this.showAlert(this.$t('INBOX_MGMT.EDIT.API.ERROR_MESSAGE'));
+      }
+    },
+    async updateWhatsAppInboxAPIKey() {
+      try {
+        const payload = {
+          id: this.inbox.id,
+          formData: false,
+        };
+
+        if (this.isAWhatsAppCloudChannel) {
+          payload.provider_config = {
```

Comment:
Do we need a conditional check here ? won't the following cover both the cases?
```
payload.provider_config = {
           api_key: this.whatsAppInboxAPIKey,
            ...this.inbox.provider_config,
          };
```

Reply:
@sojan-official There is no phone number id or business id showing in the provider_config for 360DialogWhatsAppChannel. That is the reason I added it here.